src/fiatExchanges/SelectProvider.tsx

```diff
@@ -324,6 +359,18 @@ function ExchangesSection({
 }) {
   const { t } = useTranslation()
 
+  // TODO(any): uncomment below and use params
+  // @ts-ignore
+  const { exchangesText, exchangesFlow } = useMemo(() => {
+    // if (flow === CICOFlow.CashIn) {
+    //   return getAddFundsCryptoExchangeExperimentParams()
+    // }
```

Comment:
Leaving this commented because if a version is cut before this is actually used, this will impact the experiment (unless we gate it by versions)